game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -11,11 +13,12 @@ import java.util.concurrent.{BlockingQueue, LinkedBlockingQueue}
 class GameThread(scoreThread: ScoreThread, teamIDs: List[Int]) extends Runnable {
   val MillisecondsPerTick = 16
 
+  final val actionQueue = new LinkedBlockingQueue[ActionQueryWithId]()
+  final val actionMap = new HashMap[Int, HashMap[Int, List[Action]]]()
```

Comment:
Yeah I think I agree too. If I remember correctly, it's a `HashMap `for all games that contains a `HashMap `for players with list of actions... Am I right?